app/lib/common/theme.dart

```diff
@@ -19,35 +20,128 @@ class PharmeTheme {
         onError: errorColor.highEmphasisOnColor,
         brightness: Brightness.light,
       ),
+      textTheme: textTheme,
     );
   }
 
-  static const primaryColor = MaterialColor(0xFF58A6DD, {
-    50: Color(0xFFEBF4FB),
-    100: Color(0xFFCDE4F5),
-    200: Color(0xFFACD3EE),
-    300: Color(0xFF8AC1E7),
-    400: Color(0xFF71B3E2),
-    500: Color(0xFF58A6DD),
-    600: Color(0xFF509ED9),
-    700: Color(0xFF4795D4),
-    800: Color(0xFF3D8BCF),
-    900: Color(0xFF2D7BC7),
+  static final textTheme = TextTheme(
+    titleLarge: GoogleFonts.inter(
+      fontSize: 72,
+      fontWeight: FontWeight.w300,
+      letterSpacing: -1.25,
+      color: Color(0xFF444648),
+    ),
+    titleMedium: GoogleFonts.inter(
+      fontSize: 56,
+      fontWeight: FontWeight.w300,
+      letterSpacing: -1.25,
+      color: Color(0xFF444648),
+    ),
+    titleSmall: GoogleFonts.inter(
+      fontSize: 42,
+      fontWeight: FontWeight.w300,
+      letterSpacing: -1.25,
+      color: Color(0xFF444648),
+    ),
+    headline1: GoogleFonts.inter(
+      fontSize: 32,
+      fontWeight: FontWeight.w400,
+      letterSpacing: -0.25,
+      color: Color(0xFF444648),
+    ),
+    headline2: GoogleFonts.inter(
+      fontSize: 24,
+      fontWeight: FontWeight.w400,
+      color: Color(0xFF444648),
+    ),
+    headline3: GoogleFonts.inter(
+      fontSize: 16,
+      fontWeight: FontWeight.w500,
+      color: Color(0xFF444648),
+    ),
+    headline4: GoogleFonts.inter(
+      fontSize: 12,
+      fontWeight: FontWeight.w500,
+      color: Color(0xFF444648),
+    ),
+    headline5: GoogleFonts.inter(
+      fontSize: 10,
+      fontWeight: FontWeight.w500,
+      color: Color(0xFF444648),
+    ),
+    headline6: GoogleFonts.inter(
+      fontSize: 8,
+      fontWeight: FontWeight.w500,
+      color: Color(0xFF444648),
+    ),
+    bodyText1: GoogleFonts.inter(
+      fontSize: 16,
+      fontWeight: FontWeight.w400,
+      color: Color(0xFF444648),
+    ),
+    bodyText2: GoogleFonts.inter(
+      fontSize: 12,
+      fontWeight: FontWeight.w400,
+      letterSpacing: -0.15,
+      color: Color(0xFF444648),
+    ),
+    button: GoogleFonts.inter(
+      fontSize: 16,
+      fontWeight: FontWeight.w400,
+      color: Color(0xFF444648),
+    ),
+    caption: GoogleFonts.inter(
+      fontSize: 12,
+      fontWeight: FontWeight.w400,
+      letterSpacing: -0.25,
+      color: Color(0xFF444648),
+    ),
+    overline: GoogleFonts.inter(
+      fontSize: 10,
+      fontWeight: FontWeight.w500,
+      color: Color(0xFF444648),
+    ),
+    subtitle1: GoogleFonts.inter(
+      fontSize: 24,
+      fontWeight: FontWeight.w400,
+      letterSpacing: -0.50,
+      color: Color(0xFF444648),
+    ),
+    subtitle2: GoogleFonts.inter(
+      fontSize: 16,
+      fontWeight: FontWeight.w500,
+      letterSpacing: -0.50,
+      color: Color(0xFF444648),
+    ),
```

Comment:
All of this will be deprecated pretty soon and is already not the standard for the newest material design anymore

Let's use (small, medium, large respectively) display, headline, title, body and label

https://m3.material.io/styles/typography/overview